Framework/src/TaskRunner.cxx

```diff
@@ -485,4 +489,14 @@ int TaskRunner::publish(DataAllocator& outputs)
   return objectsPublished;
 }
 
+void TaskRunner::saveToFile()
+{
+  if(!mTaskConfig.saveToFile.empty()) {
+    ILOG(Debug, Support) << "Save data to file " << mTaskConfig.saveToFile << ENDM;
+    TFile f(mTaskConfig.saveToFile.c_str(), "RECREATE");
+    mObjectsManager->getNonOwningArray()->Write();
+    f.Close();
```

Comment:
I am not sure, but maybe it would be better to save the raw objects instead of the array of MonitorObjects? I have two reasons in mind:
1. It should be easier to browse and share with others, no need to load QualityControlTypes.
2. They are saved like that in QCDB.

Reply:
indeed, I have taken the lazy path here. I checked that it worked with the TBrowser when starting root from the QC environment.

Ok, I will do it.